ts/data/sample.ts

```diff
@@ -661,9 +661,32 @@ function addQ30Contents(
     }
     return;
   }
+
+  // Create a wrapper for the contents
+  const contentWrapper = document.createElement("div");
   fragment.appendChild(
     makeMetricDisplay(sample.run.percentOverQ30, metrics, addTooltip)
   );
+
+  // Button to toggle the content
+  const toggleButton = document.createElement("button");
+  toggleButton.innerHTML = "More";
+  toggleButton.style.color = "rgb(91, 158, 90)";
+  toggleButton.classList.add("text-sm");
+  toggleButton.onclick = () => {
+    const isExpanded = contentWrapper.classList.toggle("hidden");
+    toggleButton.innerHTML = isExpanded ? "More" : "Less";
+  };
+
+  toggleButton.addEventListener("mouseover", () => {
+    toggleButton.style.textDecoration = "underline";
+  });
+  toggleButton.addEventListener("mouseout", () => {
+    toggleButton.style.textDecoration = "none";
+  });
```

Comment:
* Tailwind classes should be used for styling (colour, text decoration)
* Move button code to a separate function to reduce duplication